.build/build.ps1

```diff
@@ -92,12 +84,12 @@ task Publish.Module Build.Module, {
     Publish-Module -Name $moduleName -Repository 'PSGallery' -NuGetApiKey $ApiKey -Force -AllowPrerelease @parameters
 }
 
-task Test.Unit PreRequisites, {
+task Test.PowerShell.Unit PreRequisites, {
     Start-PesterTest -Type 'Unit'
 }
 
-task Test.Meta PreRequisites, {
-    Start-PesterTest -Type 'Meta'
+task Test.PowerShell.PsAnalyzer PreRequisites, {
```

Comment:
nit: Fwiw, I like spelling out "PowerShell" better than the "Ps". Do we need both? And if PowerShell Analyzer is a linter, maybe we just call it a lint task?

Reply:
updated